src/commands/libcoreSignAndBroadcast.js

```diff
@@ -119,7 +121,7 @@ async function signTransaction({
 
   const changePath = output ? output.getDerivationPath().toString() : undefined
   const outputScriptHex = Buffer.from(transaction.serializeOutputs()).toString('hex')
-  const lockTime = transaction.getLockTime()
+  const lockTime = undefined // TODO: transaction.getLockTime()
```

Comment:
why is this commented out? could you explain a bit what it means? wouldn't this create a regression on some coin that need it? 🤔

Reply:
seen with the libcore team: this field is not handled yet on their side, it always returns a default value, and we realized that it was what make Zcash tx fail :thinking: as it is not necessary for other coins, we choose to comment it for all, but if you think it's safer, we can set it `undefined` just for Zcash. nb: I tested on other coins (stratis, dash, ..)

Reply:
so is ZCash working right now? 🤔

Reply:
[yep](https://explorer.zcha.in/transactions/667e86a8b696a5cb60753d66ac1b900078e4e909f32da89e333afee3433984c3)

Reply:
will add a comment in the PR. hodl on